Data/OJS.Data/Migrations/20231002101437_AddProblemSubmissionTypeExecutionDetails.cs

```diff
@@ -0,0 +1,84 @@
+﻿#nullable disable
+
+namespace OJS.Data.Migrations
+{
+    using System;
+    using Microsoft.EntityFrameworkCore.Migrations;
+
+    public partial class AddProblemSubmissionTypeExecutionDetails : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropTable(
+                name: "SubmissionTypeProblems");
```

Comment:
We don't want to drop this table.. what will happen with all the data in it?

Reply:
I changed the migration to rename the table and to add time and memory limit as columns, so that the data is preserved.